### DISCUSSION TITLE
How to change terminal color

### DISCUSSION THREAD

**Discussion by mohdlatif** [Eugeny/tabby#8354] [General]
Hi, the color schemas not applies when connecting for external services.

Here is an example.

1) Fully gray ( light white ) when connecting to external server
![image](https://user-images.githubusercontent.com/56746772/236102378-fa35986c-9305-41a8-9758-0b6f8e287e1f.png)

2) Works fine for windows terminal 
![image](https://user-images.githubusercontent.com/56746772/236099292-048b65e3-cea2-451d-bdc3-1f8ae733ff11.png)

How can I make (1) have the same effect as (2)

P.S: I am using windows 

Thanks

**Comment by Eugeny** [comment]
Please see the pinned discussion in this repo

**Reply by mohdlatif** [reply]
I checked that earlier but did not find the way to **enable it in Windows**..

![image](https://user-images.githubusercontent.com/56746772/236115381-2b6946c5-fec1-4395-a389-291594d9351f.png)


How to enable it in Windows ?

Thanks

**Reply by Eugeny** [reply]
cmd.exe built-in commands don't support color output: https://serverfault.com/questions/18362/how-to-get-coloured-file-listing-in-windows-cmd-shell

**Reply by mohdlatif** [reply]
> cmd.exe built-in commands don't support color output: https://serverfault.com/questions/18362/how-to-get-coloured-file-listing-in-windows-cmd-shell

Windows 10 supports ANSI colors [since v1511](https://www.reddit.com/r/Windows10/comments/44czox/windows_10_v1511_adds_support_for_ansi_escape/)

Colors working in my CMDS

1)
![image](https://user-images.githubusercontent.com/56746772/236145421-e149b875-9b55-4420-b5c5-e5a2cc9b5a09.png)

2) 
![image](https://user-images.githubusercontent.com/56746772/236145586-cdb24e77-4121-4673-96bd-c1f451fda6bf.png)

3)
![image](https://user-images.githubusercontent.com/56746772/236099292-048b65e3-cea2-451d-bdc3-1f8ae733ff11.png)


But no color is supported when using with SSH

**Reply by medium-old** [reply]
Currently having similar issue with my SSH on Tabby, were you able to figure it out @mohdlatif ?

**Reply by mohdlatif** [reply]
No, I simply accepted its limitations and am enjoying it.

**Reply by Eugeny** [reply]
Colors are configured on the host. You need to configure colors on the host you're connecting to. See the pinned discussion.

**Reply by medium-old** [reply]
Thanks! I'm new to shell, colors aren't showing up because its through an SSH connection. I'm most interested in highlighting my own commands vs. what's returned. I thus edited my prompt command (bash) by adding the following to my ~/.bashrc file

_# User specific aliases and functions
PROMPT_COMMAND='PS1_CMD1=$(pwd)'; PS1='\[\e[38;5;211m\]\h\n\[\e[38;5;46;2;53m\]${PS1_CMD1}\[\e[0m\] \n\[\e[2;3m\]${DARAWBASE}\[\e[0m\]>'_

I'm connecting to a remote server so I can't install packages which auto-highlight syntax, but using this prompt + tabby combination makes the SSH work much easier! Also there's ways to auto-color commands such as ls :) Hope this helps!!

Links:
https://bash-prompt-generator.org/
https://mirror.apps.cam.ac.uk/pub/doc/redhat/redhat7.3/rhl-gsg-en-7.3/ls-color.html
